include/evmc/evmc.h

```diff
@@ -419,6 +419,8 @@ struct evmc_result
      *  freed with evmc_result::release().
      *
      *  This MAY be NULL.
```

Comment:
Why is this, and some other places double indented, but not every block?

Reply:
Because of migration from `///` comments to `/**` long time ago.

Reply:
Fixed.